config/routes.rb

```diff
@@ -117,4 +117,9 @@
   authenticate :user, lambda { |user| SidekiqPolicy.new(user).visible? } do
     mount Sidekiq::Web => "/sidekiq"
   end
+
+  namespace :docs do
+    mount Lookbook::Engine, at: "/components"
+  end
+  get "/lookbook", to: redirect("/docs/components")
```

Comment:
As [discussed](https://github.com/joemasilotti/railsdevs.com/issues/624#issuecomment-1236128050) I'm using `docs/components` and the route and redirecting `/lookbook` that is the default one to the new one.

Reply:
Love it – thanks for wiring this up!